lib/build-prompt.ts

```diff
@@ -184,36 +184,35 @@ function buildRetrievalText(fileItems: Tables<"file_items">[]) {
 }
 
 function adaptSingleMessageForGoogleGemini(message: any) {
-
   let adaptedParts = []
 
   let rawParts = []
-  if(!Array.isArray(message.content)) {
-    rawParts.push({type: 'text', text: message.content})
```

Comment:
Why do you touch Google Gemini code sections with a PR that addresses Groq Llama 3.1 ?